delfin/drivers/dell_emc/vmax/rest.py

```diff
@@ -407,6 +407,18 @@ def get_vmax_model(self, version=U4V_VERSION, array=''):
             vmax_version = system_info.get('model')
         return vmax_version
 
+    def get_vmax_ucode(self, version=U4V_VERSION, array=''):
```

Comment:
Is this function that can be called frequently by the client? I guess unisphere version is mostly a array details call. Can we avoid multiple get_array_detail() calls and parse and keep it with say get_vmax_model -> self.get_array_detail() #line 405

Reply:
thanks, there is no frequent calls, but still it is better to combine model and firmware get together ss both are from same REST end point.
modofied accordigly..